packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -165,14 +167,20 @@ export class SwapQuoter {
         this.orderbook = orderbook;
         this.expiryBufferMs = expiryBufferMs;
         this.permittedOrderFeeTypes = permittedOrderFeeTypes;
-        this._rfqtTakerApiKeyWhitelist = options.rfqt ? options.rfqt.takerApiKeyWhitelist || [] : [];
+        this._rfqtTakerApiKeyWhitelist = rfqt ? rfqt.takerApiKeyWhitelist || [] : [];
+        this._rfqtSkipBuyRequests =
```

Comment:
Looks like this works, but is a little confusing to read.  Does the following work? I think it reads a little easier.

```ts
const skipBuyRequests = (rfqt && rfqt.skipBuyRequests === true) || (constants.DEFAULT_SWAP_QUOTER_OPTS.rfqt && constants.DEFAULT_SWAP_QUOTER_OPTS.rfqt.skipBuyRequests === true)
```

That being said, I think we've noodled on this enough and I'm cool w/ whatever you think :P